lib/rubocop/cop/style/option_hash.rb

```diff
@@ -0,0 +1,55 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for options hashes and discourages them if the
+      # current Ruby version supports keyword arguments
+      #
+      # @example
+      #   Instead of:
+      #
+      #   def fry(options = {})
+      #     temperature = options.fetch(:temperature, 300)
+      #     ...
+      #   end
+      #
+      #   Prefer:
+      #
+      #   def fry(temperature: 300)
+      #     ...
+      #   end
+      class OptionHash < Cop
+        MSG = 'Prefer keyword arguments to options hashes.'
+
+        def on_args(node)
+          return unless supports_keyword_arguments?
+
+          last_arg = node.to_a.last
```

Comment:
I'm not sure how to write this as a splat, because I don't know how many arguments there are going to be